org.osgi.test.junit5/src/main/java/org/osgi/test/junit5/context/InstalledBundleExtension.java

```diff
@@ -63,15 +63,23 @@ public InstalledBundleExtension() {
 	 */
 	@Override
 	protected Object parameterValue(ParameterContext parameterContext, ExtensionContext extensionContext) {
-		Optional<InjectInstalledBundle> injectBundle = parameterContext.findAnnotation(supported);
+		InjectInstalledBundle injectBundle = parameterContext.findAnnotation(supported)
+			.get();
 		Parameter parameter = parameterContext.getParameter();
 		Class<?> parameterType = parameter.getType();
 		assertParameterIsOfType(parameterType, Bundle.class, supported, ParameterResolutionException::new);
-		return installedBundleOf(injectBundle.get(), extensionContext);
+		try {
+			return installedBundleOf(injectBundle, extensionContext);
+		} catch (Exception e) {
+			throw new ParameterResolutionException(
+				String.format("@InjectInstalledBundle [%s]: couldn't resolve bundle parameter [%s]: %s",
```

Comment:
```suggestion
				String.format("@%s [%s]: couldn't resolve bundle parameter [%s]: %s", supported.getSimpleName(),
```

Don't hard code annotation name and use `supported` instead.